pkgs/by-name/ro/roslyn-ls/package.nix

```diff
@@ -28,6 +29,24 @@ let
   rid = dotnetCorePackages.systemToDotnetRid stdenvNoCC.targetPlatform.system;
 
   project = "Microsoft.CodeAnalysis.LanguageServer";
+
+  targets = writeText "versions.targets" ''
+    <Project>
+      <ItemGroup>
+        <KnownFrameworkReference Update="@(KnownFrameworkReference)">
```

Comment:
I'm not happy with how verbose this is.  I wouldn't want a lot of packages to have to do this sort of thing, but it's going to take some time to deal with this in the sdk packages.